pkg/apis/consts/consts.go

```diff
@@ -0,0 +1,11 @@
+package consts
+
+const (
+	NetworkPluginAzure     = "azure"
+	NetworkPluginKubenet   = "kubenet"
+	
+	PodNetworkTypeOverlay  = "overlay"
+	PodNetworkTypeNone = ""
+
+	NetworkDataplaneCilium = "cilium"
```

Comment:
We could also potentially get these from the Azure sdk for go. IIRC they are exposed in the AKS API.